File: LEETCODE/1486. 数组异或操作.java
class Solution {
    public int xorOperation(int n, int start) {
        int[] nums = new int[n];
        int res = 0;
        for (int i=0;i<n;i++) {
            nums[i] = start + 2*i;
            res ^= nums[i];
        }
        return res;
    }
}

File: LEETCODE/kotlin/1486. 数组异或操作.kt
class Solution {
    fun xorOperation(n: Int, start: Int): Int {
        var nums = IntArray(n)
        var res = 0
        for (i in 0..n-1) {
            nums[i] = start + 2*i
            res = res xor nums[i]
        }
        return res
    }
}
